specification/riskiq/data-plane/Microsoft.Easm/preview/2022-04-01-preview/easm.json

```diff
@@ -0,0 +1,5041 @@
+      "properties": {
+        "completedDate": {
+          "description": "The date for when the disco run was completed in the microservice.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "excludes": {
+          "description": "The list of excludes used for the disco run, aka assets to exclude from the discovery algorithm.",
+          "items": {
+            "$ref": "#/definitions/AssetId"
+          },
+          "type": "array"
+        },
+        "newAssetsFoundCount": {
+          "description": "The count of the new assets that were found this disco run, aka that are labeled with this runs id at this time",
+          "format": "int64",
+          "type": "integer"
+        },
+        "seeds": {
+          "description": "The list of seeds used for the disco run.",
+          "items": {
+            "$ref": "#/definitions/AssetId"
+          },
+          "type": "array"
+        },
+        "startedDate": {
+          "description": "The date for when the disco run was actually started in the microservice.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "state": {
+          "description": "The State of the disco run",
+          "enum": [
+            "pending",
+            "running",
+            "completed",
+            "failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "discoRunResponseState"
+          }
+        },
+        "submittedDate": {
+          "description": "The date for when the disco run was created in the microservice.",
+          "format": "date-time",
+          "type": "string"
+        },
+        "tier": {
+          "description": "The tier which will affect the algorithm used for the disco run.",
+          "type": "string"
+        },
+        "totalAssetsFoundCount": {
+          "description": "The total count of assets that were found this disco run.",
+          "format": "int64",
+          "type": "integer"
+        }
+      },
+      "type": "object"
+    },
+    "DiscoTemplatePageResponse": {
+      "properties": {
+        "content": {
+          "items": {
+            "$ref": "#/definitions/DiscoTemplateResponse"
+          },
+          "type": "array"
+        },
+        "first": {
+          "type": "boolean"
+        },
+        "last": {
+          "type": "boolean"
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "number": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "numberOfElements": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "size": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "totalElements": {
+          "format": "int64",
+          "type": "integer"
+        },
+        "totalPages": {
+          "format": "int32",
+          "type": "integer"
+        }
```

Comment:
Why were all the descriptions removed?

Reply:
Seems quite a few descriptions were removed from various models below.

Reply:
Same as above.  Fixed in the 2022-09-01-preview version.